web/react/components/edit_post_modal.jsx

```diff
@@ -59,6 +61,18 @@ export default class EditPostModal extends React.Component {
     handleUserInput(e) {
         this.setState({editText: e.target.value});
     }
+    handleEditPostEvent(options) {
+        this.setState({
+            editText: options.message || '',
+            title: options.title || '',
+            post_id: options.postId || '',
```

Comment:
Yep should be lastPostId, don't know why it worked for me probably some caching in there, I'm going to fix it later this eve